abel/linbasex.py

```diff
@@ -0,0 +1,418 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import numpy as np
+import abel
+import scipy as sci
```

Comment:
I would remove this line. In general, there is no need to "import scipy", since, by itself, doesn't really load anything useful. And I would rather just see at a scipy instead of sci. 

You'll need to change "sci" to "scipy" in a few other lines in this file.